### ISSUE TITLE
Password with MacBook

### ISSUE THREAD

**Issue by philippe44** [postlund/pyatv#2347]
### What do you need help with?

Hi - I have a user of my AirPlay-based apps that wants to use his macbook pro as a player. That works without password, but he'd like to setup one. Unfortunately, the password implementation I did, that works with other speakers (I'm not talking about pairing of MFi) does not seem to work, and there is no "pw" key in the TXT record. Are you aware of that macOS password mode? Thanks

**Comment by postlund** [comment]
Yeah, password is only implemented for RAOP and not AirPlay at the moment. Shouldn't be that hard to fix since it's basically the same code, but it needs to be done. Definitely technical debt at this point that should be dealt with.

**Comment by postlund** [comment]
#556 for reference
